dot/state/db.go

```diff
@@ -44,7 +44,7 @@ func LoadBestBlockHash(db database.Database) (common.Hash, error) {
 
 // StoreGenesisData stores the given genesis data at the known GenesisDataKey.
 func StoreGenesisData(db database.Database, gen *genesis.Data) error {
-	enc, err := scale.Encode(gen)
+	enc, err := json.Marshal(gen)
 	if err != nil {
```

Comment:
This is great, I was stuck here because I couldn't figure out how to scale encode/decode type make[string]interface{}, but of course it doesn't need to be scale encode, it can be JSON.  Nice!